lms/djangoapps/courseware/tabs.py

```diff
@@ -35,6 +36,16 @@ class CoursewareTab(EnrolledTab):
     is_movable = False
     is_default = False
 
+    @property
+    def link_func(self):
+        """
+        Returns a function that computes the URL for this tab.
+        """
+        if waffle.switch_is_active('unified_course_view'):
```

Comment:
Reviewers: we discussed at Onyx stand-up that we can't use a Waffle flag as that requires a request object. It might be possible to rework the code to pass it through, but for our use case it seems enough just to have a single switch.